image-processing/linefinder.py

```diff
@@ -192,159 +60,156 @@ def find_lines_with_exclusions(self, view_plot = True, distance = False, min_pro
 
         '''
         
-        blurred = linefinder.blur_sample_gauss(self, False)
-        peaks_no_exclusions = linefinder.scipy_peaks(self, False)
-        prominences_no_exclusions = linefinder.find_prominences(self, False)
-        
-        if distance and min_prominences == False:
-            raise ValueError('Please chose at least one exclusion, either distance or prominence')
-        
-        prominences = None      #these are set to none in case they aren't set in the inputs by the user 
-        min_distance = None 
-
-
-        if isinstance(min_prominences, bool):
-            if not min_prominences: #if no value is given, then it takes prominences that are greater than the mean only 
+        if not min_prominences: #if no value is given, then it takes prominences that are greater than the mean only 
 
             #it is probably sensible to also add a minimum prominence here, but more research is needed to find what this minimum should be 
                 
-                mean_prominence = np.mean(prominences_no_exclusions)
-                prominences = prominences_no_exclusions[prominences_no_exclusions>=mean_prominence]
-                print(prominences)
+            self.min_prominences = self.prominences[self.prominences>=self.mean_prominence]
```

Comment:
self.mean_prominence doesn't exist, I guess it's meant to be mean_prominences?

Reply:
corrected (TF)